daprdocs/content/en/developing-applications/building-blocks/state-management/state-store-ttl.md

```diff
@@ -31,61 +31,46 @@ Please refer to the TTL column in the tables at [state store components]({{< ref
 
 State TTL can be set in the metadata as part of the state store set request:
 
-{{< tabs "HTTP API (Bash)" "HTTP API (PowerShell)" "Python SDK" "PHP SDK">}}
+{{< tabs Python "HTTP API (Bash)" "HTTP API (PowerShell)">}}
```

Comment:
only Python example available?

Reply:
Yes. I could not find the example for other languages.